src/videotile/DefaultVideoTile.ts

```diff
@@ -131,7 +131,8 @@ export default class DefaultVideoTile implements DevicePixelRatioObserver, Video
     mediaStream: MediaStream | null,
     contentWidth: number | null,
     contentHeight: number | null,
-    streamId: number | null
+    streamId: number | null,
+    externalUserId?: string | null
```

Comment:
Why is this parameter optional?

Reply:
To make sure that the API already being used by customers do not break.